### PR TITLE
fix: prevent injection of scripts over and over

### PR DESCRIPTION
**Pull request by omBratteng**
😢

### CODE REVIEW

File: packages/extension/src/lib/extensionScripts.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const HOST_PERMISSIONS = [
   'https://*.daily.dev/*',
 ];
 
-const hasInjectedScripts = false;
+let hasInjectedScripts = false;
```
Comment (rebelchris) [inline packages/extension/src/lib/extensionScripts.ts:23]: This even looks like it was meant to do something like this? 🤔

**Review by rebelchris — APPROVED** [review]
Worth a shot by now.

File: packages/extension/src/lib/extensionScripts.ts (hunk 1 above)
Comment (omBratteng) [reply packages/extension/src/lib/extensionScripts.ts:23]: Yeah. It did it originally
https://github.com/dailydotdev/apps/blob/111feb347c5cd312a6cc6d75a54043005dbd5a3c/packages/extension/src/lib/extensionScripts.ts#L26-L35

Comment (omBratteng) [reply packages/extension/src/lib/extensionScripts.ts:23]: Probably lint auto-fix that saw a let that wasn't re-written and converted it to a const. That's what happens locally with me if I remove the `hasInjectedScripts = true;`, then it's convert to `const hasInjectedScripts = false;`.

Comment (omBratteng) [reply packages/extension/src/lib/extensionScripts.ts:23]: And then no-one caught the change

**Review by DragosIorgulescu — APPROVED** [review]
Great work on this investigation, and sorry for introducing this issue 🤦‍♂️

File: packages/extension/src/lib/extensionScripts.ts [hunk 2]
```diff
@@ -53,6 +53,7 @@ export const registerBrowserContentScripts = async (): Promise<void> => {
       },
     ]);
   }
+  hasInjectedScripts = true;
```
Comment (omBratteng) [inline packages/extension/src/lib/extensionScripts.ts:56]: Could probably remove this one.

Comment (omBratteng) [reply packages/extension/src/lib/extensionScripts.ts]: Removed it, didn't do anything really.

**Review by kopancek — APPROVED** [review]
Great catch!

**Pull request merged**